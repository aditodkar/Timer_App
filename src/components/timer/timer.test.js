import React from 'react';
import renderer from 'react-test-renderer';
import Timer from './timer';


it('Timer component snapshot', () => {
    const snap = renderer.create(<Timer/>).toJSON();
    expect(snap).toMatchSnapshot();
});
