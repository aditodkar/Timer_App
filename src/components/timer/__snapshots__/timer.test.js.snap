// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Timer component snapshot 1`] = `
<div>
  <span
    className="unit"
  >
    NaN
    :
  </span>
  <span
    className="unit"
  >
    NaN
    .
  </span>
  <span
    className="unit"
  >
    N
  </span>
</div>
`;
